openmc/deplete/pool.py

```diff
@@ -30,14 +32,17 @@ def deplete(func, chain, x, rates, dt, matrix_func=None):
     rates : openmc.deplete.ReactionRates
         Reaction rates (from transport operator)
     dt : float
-        Time in [s] to deplete for
+        Time in seconds to deplete for
```

Comment:
Writing it as `[units]` is the convention that we use in our documentation whenever there is a quantity with units
```suggestion
        Time in [s] to deplete for
```